lib/rex/post/smb/ui/console/command_dispatcher/shares.rb

```diff
@@ -181,12 +182,31 @@ def cmd_ls(*args)
             print_line table.to_s
           end
 
+          #
+          # Alias the ls command to dir, for those of us who have windows muscle-memory
+          #
+          alias cmd_dir cmd_ls
+
+          def cmd_ls_help
+            print_line 'Usage:'
+            print_line 'ls [options] [glob/path]'
+            print_line
+            print_line 'COMMAND ALIASES:'
+            print_line
+            print_line '    dir'
+            print_line
+            print_line 'Lists contents of directory or file info, searchable'
+            print_line @@ls_opts.usage
+          end
+
           def cmd_ls_tabs(_str, words)
             return [] if words.length > 1
 
             @@ls_opts.option_keys
           end
 
+          alias cmd_dir_tabs cmd_ls_tabs
+
```

Comment:
I think we need to alias the help menu for dir too

Reply:
This is what I'm seeing in the meterpreter implementation at least

```
  #
  # Alias the ls command to dir, for those of us who have windows muscle-memory
  #
  alias :cmd_dir :cmd_ls
  alias :cmd_dir_help :cmd_ls_help
  alias :cmd_dir_tabs :cmd_ls_tabs
```

Reply:
So I tested `dir -h` and it worked as expected. I had originally lifted those one by one to test if I needed them and that is why I had left that line out for the help commands. Was `dir -h` the correct test for that?

Reply:
I believe it's `help dir`

Reply:
Just tested that and you're right, I have updated the code to included that third alias.